odoo/models.py

```diff
@@ -4323,18 +4323,21 @@ def _check_company(self, fnames=None):
             lines = [_("Incompatible companies on records:")]
             company_msg = _lt("- Record is company “%(company)s” and “%(field)s” (%(fname)s: %(values)s) belongs to another company.")
             record_msg = _lt("- “%(record)s” belongs to company “%(company)s” and “%(field)s” (%(fname)s: %(values)s) belongs to another company.")
+            record_companies_msg = _lt("- “%(record)s” belongs to companies %(company)s but “%(field)s” (%(fname)s: %(values)s) belongs to another company.")
```

Comment:
Also I don't really understand the why we use `_lt` here 🤔

Reply:
I agree it makes sense to change it to `_`. :smile: